internal/attestation/azure/snp/testdata/testdata.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+// Package testdata contains testing data for an attestation process.
+package testdata
```

Comment:
This is not guarded by tags. Does that mean that we also embed those files in release binaries or is go smart enough here to see that they are unused?

Reply:
How would I tag the file so that it get's included when running unit tests?

Reply:
As long as this package isn't imported by any files other than *_test.go, it won't be included in release binaries.